db/migrate/20241017215227_add_columns_to_state_file_w2.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnsToStateFileW2 < ActiveRecord::Migration[7.1]
+  def change
+    add_column :state_file_w2s, :employer_name, :string
+    add_column :state_file_w2s, :employee_name, :string
```

Comment:
as discussed, these should never be `null` -- `EmployeeNm` and `EmployerName` seem to be required fields on the federal IRS schema